data/tlg0062/tlg001/tlg0062.tlg001.perseus-eng4.xml

```diff
@@ -32,19 +32,18 @@
 <listBibl xml:lang="eng">
 <biblStruct>
 <monogr>
+    <author>Lucian</author>
+    <title xml:lang="eng">The works of Lucian of Samosata</title>
```

Comment:
extra language tag here but no big deal

Reply:
I will fix that! And then merge this! I will be doing a number more of these over next week or two but will try not to send too many files at atime.